build/common.targets

```diff
@@ -11,6 +11,11 @@
     <IsCore>true</IsCore>
   </PropertyGroup>
 
+  <PropertyGroup Condition=" '$(TargetFramework)' == 'netcoreapp3.0' ">
+    <DefineConstants>$(DefineConstants);IS_NETCORE30</DefineConstants>
+    <IsCore30>true</IsCore30>
```

Comment:
You don't need this, since it will be used only in 1 more place.

Reply:
Thanks! Changed accordingly.